### PR TITLE
Remove unused or depreciated agreement fields

### PR DESCRIPTION
**Pull request by C-gyorfi**
## Context
There are many agreement related fields pulled when syncing case priority from UH. These fields are also getting passed to the front-end when calling `ViewCases` use case. Going forward we want to use agreement data from MA. 

## Changes proposed in this pull request
- Remove references of unused/depreciated agreements fields 🧹  🗑️ 

## Things to check
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] Environment variables have been updated

### CODE REVIEW

File: spec/lib/hackney/income/view_cases_spec.rb [hunk 1]
```diff
@@ -130,14 +119,8 @@
                                              current_arrears_agreement_status: nil,
 
                                              # TODO: the following fields needs update, no longer should be pulled from UH agreements
```
Comment (antonyoneill) [inline spec/lib/hackney/income/view_cases_spec.rb:121]: Can these comments be removed now?

File: spec/lib/hackney/income/view_cases_spec.rb [hunk 2]
```diff
@@ -154,16 +137,9 @@
           expect(subject.cases).to include(a_hash_including(
                                              ref: tenancy_attributes.fetch(:ref),
                                              current_arrears_agreement_status: agreements.last.current_state,
-
                                              # TODO: the following fields needs update, no longer should be pulled from UH agreements
```
Comment (antonyoneill) [inline spec/lib/hackney/income/view_cases_spec.rb:140]: Can these comments be removed now?

**Review by antonyoneill — APPROVED** [review]
This looks OK

I think `broken_court_order` was being serialised in the tenancies_controller before hand, did you check this is not being used in the UI? (I suspect you have, just want to double check)

File: spec/lib/hackney/income/view_cases_spec.rb (hunk 1 above)
Comment (C-gyorfi) [reply spec/lib/hackney/income/view_cases_spec.rb:121]: Working on this, I just wanted to finish my day with a PR :D

**Pull request merged**